0000-template.md

```diff
@@ -1,23 +1,31 @@
 ---
-Stage: Accepted
-Start Date: 
-Release Date: Unreleased
-Release Versions:
+stage: accepted
+start-date: 
+release-date: 
+release-versions:
   ember-source: vX.Y.Z
   ember-data: vX.Y.Z
-Relevant Team(s): 
-RFC PR: 
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+proposal-pr:
```

Comment:
I think we will probably want to have a bit of metadata for each stage's PR

Reply:
Good point!

Reply:
this is now done 👍